src/flowMC/resource_strategy_bundle/RQSpline_GRW.py

```
@@ -40,14 +41,14 @@ def __init__(
         rng_key: Key,
         n_chains: int,
         n_dims: int,
-        logpdf: Callable[[Float[Array, " n_dim"], dict], Float],
+        logpdf: Callable[[Float[Array, " n_dim"], dict], FloatScalar],
```

Ruff (0.15.18)

[error] Syntax error in forward annotation: Expected an expression

(F722)